### PR TITLE
DOCS-1499: Document `dual-gps-rtk` model of movement sensor

### PR DESCRIPTION
**Pull request by sguequierre**
* Adds new model

### CODE REVIEW

File: docs/components/movement-sensor/_index.md [hunk 1]
```diff
@@ -48,6 +48,7 @@ Model | Description <a name="model-table"></a>
 [`gps-nmea`](./gps/gps-nmea/) | [NMEA-based](https://en.wikipedia.org/wiki/NMEA_0183) GPS models
 [`gps-nmea-rtk-pmtk`](./gps/gps-nmea-rtk-pmtk/) | [NTRIP-based](https://en.wikipedia.org/wiki/Networked_Transport_of_RTCM_via_Internet_Protocol) [RTK](https://en.wikipedia.org/wiki/Real-time_kinematic_positioning) GPS models using I<sup>2</sup>C (**experimental**)
 [`gps-nmea-rtk-serial`](./gps/gps-nmea-rtk-serial/) | [NTRIP-based](https://en.wikipedia.org/wiki/Networked_Transport_of_RTCM_via_Internet_Protocol) [RTK](https://en.wikipedia.org/wiki/Real-time_kinematic_positioning) GPS models using serial communication (**experimental**)
+[`dual-gps-rtk`](./dual-gps-rtk/) | A movement sensor that calculates compass heading from two GPS movement sensors
```
Comment (sguequierre) [inline docs/components/movement-sensor/_index.md:51]:
```suggestion
[`dual-gps-rtk`](./gps/dual-gps-rtk/) | A movement sensor that calculates compass heading from two GPS movement sensors
```

File: docs/components/movement-sensor/gps/dual-gps-rtk.md [hunk 2]
```diff
@@ -0,0 +1,108 @@
...
+title: "Configure a Dual GPS Movement Sensor"
+linkTitle: "dual-gps-rtk"
+weight: 10
+type: "docs"
+description: "Configure a movement sensor that calculates compass heading from two gps movement sensors."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `dual-gps-rtk` model of movement sensor calculates compass heading from two gps movement sensors.
```
Comment (randhid) [inline docs/components/movement-sensor/gps/dual-gps-rtk.md:11]:
```suggestion
The `dual-gps-rtk` model of movement sensor calculates compass heading from two gps movement sensors, and returns the midpoint Position between the first an second GPS devices as its position.
```

File: docs/components/movement-sensor/gps/dual-gps-rtk.md [hunk 3]
```diff
@@ -0,0 +1,108 @@
...
+In addition to [`CompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`GetPosition()`](/components/movement-sensor/#getposition).
```
Comment (randhid) [inline docs/components/movement-sensor/gps/dual-gps-rtk.md:12]:
```suggestion
In addition to [`CompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`Position()`](/components/movement-sensor/#getposition).
```

Either use `Get` as a prepend for both or don't. The `go` APIs follow the go style of not prefixing `Get` to the API function names, the other SDKs do.

File: docs/components/movement-sensor/gps/dual-gps-rtk.md [hunk 4]
```diff
@@ -0,0 +1,108 @@
...
+
+{{% /tab %}}
+{{< /tabs >}}
+
+The following attributes are available for a `dual-gps-rtk` movement sensor:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `first_gps` | int | **Required** | The name you have configured for the first movement sensor you want to combine the measurements from. Must be a `GPS` model. |
```
Comment (randhid) [inline docs/components/movement-sensor/gps/dual-gps-rtk.md:104]: Technically we're only checking if the movement sensor is position reporting in the code, but I doubt this model will be useful for non-GPS position reporting sensors such as wheeled odometry, SLAM algorithms or optical trackers.

**Review by randhid — APPROVED** [review]
Few nits, looks good to me! sorry it took so long, I was on vacation!

File: docs/components/movement-sensor/gps/dual-gps-rtk.md (hunk 3 above)
Comment (sguequierre) [inline docs/components/movement-sensor/gps/dual-gps-rtk.md:12]:
```suggestion
In addition to [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`GetPosition()`](/components/movement-sensor/#getposition).
```

Comment (sguequierre) [reply docs/components/movement-sensor/gps/dual-gps-rtk.md:12]: Sounds good-- will use Get to match docs I think

File: docs/components/movement-sensor/gps/dual-gps-rtk.md (hunk 4 above)
Comment (sguequierre) [reply docs/components/movement-sensor/gps/dual-gps-rtk.md:104]: Should I specify position reporting then, or leave as is?

Comment (randhid) [reply docs/components/movement-sensor/gps/dual-gps-rtk.md:104]: Leaving as is is more useful.

File: docs/components/movement-sensor/gps/dual-gps-rtk.md [hunk 5]
```diff
@@ -0,0 +1,108 @@
...
+linkTitle: "dual-gps-rtk"
+weight: 10
+type: "docs"
+description: "Configure a movement sensor that calculates compass heading from two gps movement sensors."
+images: ["/icons/components/imu.svg"]
+# SMEs: Rand
+---
+
+The `dual-gps-rtk` model of movement sensor calculates compass heading from two GPS movement sensors, and returns the midpoint position between the first and second GPS devices as its position.
+In addition to [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`GetPosition()`](/components/movement-sensor/#getposition).
```
Comment (sguequierre) [inline docs/components/movement-sensor/gps/dual-gps-rtk.md:12]:
```suggestion
In addition to [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading), this model provides data for [`GetPosition()`](/components/movement-sensor/#getposition) and [`GetAccuracy()`](/components/movement-sensor/#getaccuracy).
```
@randhid would this be accurate?

File: docs/components/movement-sensor/gps/dual-gps-rtk.md (hunk 4 above)
Comment (sguequierre) [reply docs/components/movement-sensor/gps/dual-gps-rtk.md:104]: Ok, thanks!

File: docs/components/movement-sensor/gps/dual-gps-rtk.md (hunk 5 above)
Comment (randhid) [reply docs/components/movement-sensor/gps/dual-gps-rtk.md:12]: yep

File: docs/components/movement-sensor/gps/dual-gps-rtk.md [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+{{< /tabs >}}
+
+The following attributes are available for a `dual-gps-rtk` movement sensor:
+
+<!-- prettier-ignore -->
+| Name | Type | Inclusion | Description |
+| ---- | ---- | --------- | ----------- |
+| `first_gps` | int | **Required** | The name you have configured for the first movement sensor you want to combine the measurements from. Must be a `GPS` model. |
+| `second_gps` | string | **Required** | The name you have configured for the second movement sensor you want to combine the measurements from. Must be a `GPS` model. |
+| `offset_degrees` | int | Optional | Value to offset the compass heading calculation between the two GPS devices based on their positions on the base. Calculate this as the degrees between the vector from `first_gps` to `second_gps` and the vector from the vehicle's back to the vehicle's front, counterclockwise. {{< imgproc src="/components/movement-sensor/offset_degrees.png" alt="Rand's diagram of 3 offset degree calculations." resize="600x" >}} <br> Default: `90` |
```
[the remark this answers is not in the document]
Comment (sguequierre) [reply docs/components/movement-sensor/gps/dual-gps-rtk.md:106]: Thank you!

**Pull request merged**